uvicorn/main.py

```diff
@@ -149,6 +149,13 @@ def get_logger(log_level):
     help="Close Keep-Alive connections if no new data is received within this timeout.",
     show_default=True,
 )
+@click.option(
+    "--enable-lifespan",
+    is_flag=True,
+    default=False,
```

Comment:
We should default to using lifespan, rather than having it off by default.
I'm not sure how that would work with click's flags, or what we should use as the option for disabling support